app-admin/etcd-wrapper/files/etcd-member.service

```diff
@@ -0,0 +1,26 @@
+[Unit]
+Description=etcd (System Application Container)
+Documentation=https://github.com/coreos/etcd
+Conflicts=etcd.service
+Conflicts=etcd2.service
+
+[Service]
+Type=notify
+Restart=always
+RestartSec=10s
+LimitNOFILE=40000
+TimeoutStartSec=60
+
+Environment="ETCD_IMG_TAG=v3.0.7"
```

Comment:
We are going to use whichever version the etcd team suggests. Since there is a default (which will change as the OS updates), etcd will always need to be able to migrate forward.